airbyte-webapp/src/pages/OnboardingPage/OnboardingPage.tsx

```diff
@@ -91,6 +98,14 @@ const OnboardingPage: React.FC = () => {
     connectionConfiguration?: any;
   }) => {
     setErrorStatusRequest(0);
+
+    const sourceConnector = getSourceById(values.serviceType);
+    AnalyticsService.track("New Source - Action", {
+      user_id: config.ui.workspaceId,
+      action: "Test a connector",
+      connector_source: sourceConnector?.name
```

Comment:
Can we also log the uuid?

Reply:
(assuming it is available)

Reply:
In general, all the places where we are logging a name, I wonder if we can also log the associated uuid

Reply:
Not quite sure what uuid you want to log here. Id of user? I do not see any other ids in tracking plan

Reply:
@johnlafleur can you update the logging plan to also include the source/desination uuids? Names are not a reliable way to track as they usually change.

Reply:
Added